index.d.ts

```diff
@@ -0,0 +1,31 @@
+declare module 'resin-jwt' {
+	import Strategy from 'passport-jwt';
```

Comment:
Does `passport-jwt` have typings by default or do we need to add the typings to package.json?